scripts.js

```diff
@@ -225,6 +224,95 @@ window.adobeid = {
   locale: window.blog.language,
 };
 
+/**
+ * Set up a click event on Region Picker
+ */
+function handleDropdownRegion() {
+  const currentLocale = window.blog.language;
+  const localeNames = {
+      'ko': 'Korea (한국어)',
+      'en': 'APAC (English)',
+      'en': 'USA (English)',
+      'en': 'UK (English)',
+  };
+
+  // Change locale value from Feds Region Picker Button
+  const FEDSregionPickerText = document.querySelector('.feds-regionPicker-text');
```

Comment:
This line is executed on `load`, but FEDS might not be ready by then and the element might not be available in the DOM; you can check for this by using the following code:
if (typeof feds === 'object' and typeof feds.events === 'object' and feds.events.experience === true) {
    // FEDS is loaded, the DOM element is available
    handleDropdownRegion();
} else {
    // FEDS is not ready, wait for it to be ready
    window.addEventListener('feds.events.experience.loaded', handleDropdownRegion);
}